pages/oai/OAIHandler.inc.php

```diff
@@ -39,13 +41,23 @@ function index($args, $request) {
 
 	/**
 	 * Validate the request
+	 * @param $request PKPRequest
 	 */
-	function validate() {
+	function validate($request) {
```

Comment:
Should the argument signature match `PKPHandler::validate`? https://github.com/pkp/pkp-lib/blob/stable-3_1_2/classes/handler/PKPHandler.inc.php#L318